### PR TITLE
added basic layout and index, about page content

### PR DESCRIPTION
**Pull request by rajr25859**

### CODE REVIEW

File: templates/_layout.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            </div>
+        </nav>
+    </header>
+
+    <main class="container" style="padding-top: 10px;">
+        {% block content %}{% endblock %}
+    </main>
+
+    <footer>
+
```
Comment (soni0577) [inline templates/_layout.html]: Please add Some Content in Footer... at least Copyright statement.

File: templates/about.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% extends '_layout.html' %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col">
+        <div class="jumbotron">
+            <h1 class="display-4">About Us</h1>
+            <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention
```
Comment (soni0577) [inline templates/about.html]: Please add a list of Contributors here

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends '_layout.html' %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col">
+        <div class="jumbotron">
+            <h1 class="display-4">Welcome to Blog App!</h1>
+            <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention
```
Comment (soni0577) [inline templates/index.html]: Change this dummy text

File: templates/_layout.html (hunk 1 above)
Comment (rajr25859) [reply templates/_layout.html]: done

File: templates/index.html (hunk 3 above)
Comment (rajr25859) [reply templates/index.html]: this is a part of different task.

File: templates/about.html (hunk 2 above)
Comment (rajr25859) [reply templates/about.html]: done!

**Comment by rajr25859** [comment]
@soni0577 please verify your change requests

**Comment by soni0577** [comment]
All done. everything looks fine. Please merge.

**Pull request merged**